message/message-util/util/src/java/org/sakaiproject/message/util/BaseMessage.java

```diff
@@ -3916,6 +3916,12 @@ public Time getDate()
 
 		} // getDate
 		
+
+		@Override
+		public Instant getInstant() {
+			Instant i = Instant.ofEpochMilli(m_date.getTime());
+			return i;
+		}
 		/**
```

Comment:
Why not just return it without the variable?

Reply:
OK